components/overflow-group/overflow-group-mixin.js

```diff
@@ -25,7 +23,7 @@ async function filterAsync(arr, callback) {
 	return results.filter(i => i !== fail);
 }
 
-export const OverflowGroupMixin = superclass => class extends LocalizeCoreElement(RtlMixin(superclass)) {
+export const OverflowGroupMixin = superclass => class extends LocalizeCoreElement(superclass) {
```

Comment:
I checked consumers extending `OverflowGroupMixin` to make sure they were not assuming `RtlMixin` was present.